packages/core/test/invoke.test.ts

```diff
@@ -2423,7 +2446,7 @@ describe('invoke', () => {
         .start();
     });
 
-    it('should sync with child machine when sync: true option is provided', (done) => {
+    it.skip('should sync with child machine when sync: true option is provided', (done) => {
```

Comment:
IMHO we should completely remove the `sync` option and the associated `xstate.update` event. We can defer the decision about this but then we should add a TODO comment here

Reply:
Yes, I agree